cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -51,30 +54,41 @@ started correctly.
      # Run this command to configure your shell: 
      # eval $(docker-machine env cdap)
  
-   It is essential to run these export commands (or the single ``eval`` command). Otherwise,
-   subsequent Docker commands will fail because they won't be able to connect to the correct
-   Docker VM.
+   It is essential to run these export commands (or the single ``eval`` command, ``eval
+   $(docker-machine env cdap)``). Otherwise, subsequent Docker commands will fail because
+   they won't be able to connect to the correct Docker VM.
   
 #. If you are running **Docker on either Mac OS X or Microsoft Windows**, Docker is running a
    virtual Linux machine on top of your host OS. You will need to use the address shown
    above (such as ``192.168.99.100``) as the host name when either connecting to the CDAP
    UI or making an HTTP request.
 
+#. If you are running **Docker on Microsoft Windows**, note that paths used employ Linux's
+   forward-slashes (``/``) and not back-slashes (``\``).
+
 #. Once Docker has started, pull down the *CDAP Docker Image* from the Docker Hub using:
 
    .. tabbed-parsed-literal::
      :tabs: "Linux or Mac OS X",Windows
      :mapping: linux,windows
      :dependent: linux-windows
      :languages: console,shell-session
- 
-     .. Linux or Mac OS X
-     
+     :keepslashes:
+
      $ docker pull caskdata/cdap-standalone:|release|
 
-     .. Windows
-    
-     > docker pull caskdata/cdap-standalone:|release|
+#. Start the *CDAP Standalone* Docker container with:
+
+   .. tabbed-parsed-literal::
+     :tabs: "Linux or Mac OS X",Windows
+     :mapping: linux,windows
+     :dependent: linux-windows
+     :languages: console,shell-session
+     :keepslashes:
+ 
+     $ docker run -d --name cdap-standalone -p 11011:11011 -p 11015:11015 caskdata/cdap-standalone:|release|
```

Comment:
I missed this, we should use `--name cdap-sdk` since that's what we're using below.